spec/system/grant_submissions/grant_submission_responses_spec.rb

```diff
@@ -11,69 +15,115 @@
       find_field('Your Project\'s Title', with: '').set(Faker::Lorem.sentence)
     end
 
-    scenario 'requires response mandatory short_text question' do
-      short_text_question = @grant.questions.where(response_type: 'short_text').first
-      short_text_question.update_attribute(:is_mandatory, true)
+    context 'valid submission' do
+      scenario 'accepts a pick_one response' do
+        multiple_choice_question = @grant.questions.where(response_type: 'pick_one').first
```

Comment:
change to pick_one and other places in the file.

Reply:
agreed. will move to its own ticket, though.

Reply:
See #290